gyp/pylib/gyp/common.py

```diff
@@ -342,12 +342,12 @@ def __init__(self):
       # consequently ending up with current dir "/cygdrive/c/..." being prefixed to those, which was
       # obviously a non-existent path, for example: "/cygdrive/c/<some folder>/C:\<my win style abs path>".
       # See https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp for more details
-      baseTempDir = "" if IsCygwin() else os.path.split(filename)[0];
+      base_temp_dir = "" if IsCygwin() else os.path.split(filename)[0];
```

Comment:
Can we please lose the trailing semicolon?  This ain't JavaScript.

Reply:
Thanks, it's a habit from 10+ years working with Java and JavaScript.

Reply:
Corrected